scripts/build.mjs

```diff
@@ -0,0 +1,43 @@
+import { copyFile, readFile, writeFile, rm } from "node:fs/promises";
+
+const sharedOptions = {
+	sourcemap: "external",
+	sourcesContent: true,
+	minify: false,
+	allowOverwrite: true,
+	packages: "external",
+};
+
+async function main() {
```

Comment:
Indentation needs fixing

Reply:
`npm run lint:fix` reports correct indentation for me.